Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -28,6 +29,15 @@ public final class RootTabBarViewController: UITabBarController {
     super.viewDidLoad()
     self.delegate = self
 
+    self.applicationWillEnterForegroundObserver = NotificationCenter
+      .default
+      .addObserver(
+        forName: UIApplication.willEnterForegroundNotification,
```

Comment:
I figured observing this notification was better than having to pass this through from `AppDelegate`. It's the same event.